### DISCUSSION TITLE
DietPi-Software | Scrypted home automation platform

### DISCUSSION THREAD

**Discussion by fmunteanu** [MichaIng/DietPi#5422] [Software Request] [answered]
#### Is your feature request related to a problem?
No.

#### Describe the solution you'd like:
[Scrypted](https://github.com/koush/scrypted) is a home automation platform primarily focusing on making camera experiences seamless. It supports Google Home, Apple HomeKit and Amazon Alexa. List of supported plugins (I'm interested in Unifi Protect plugin):
https://github.com/koush/scrypted/tree/main/plugins

[Installation](https://github.com/koush/scrypted/wiki/Installation:-Linux) should be fairly easy to implement into Dietpi software ecosystem, since most of the components are already installed.

#### Describe alternatives you've considered:
Currently there is a HomeBridge Pi image and Unifi Protect plugin. However, while testing Scrypted, it is much more stable and responsive.

**Comment by MichaIng** [comment, marked as answer]
Implemented with DietPi v10.7.
